addons/move-to-top-layer/addon.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name":"Move Sprite to Top Layer",
+  "description":"This addon allows you to move a selected sprite to the top layer",
```

Comment:
There could be a little more information regarding what the top layer means. Maybe "the front (top layer) of the stage"?